### DISCUSSION TITLE
Usage with Snowpack

### DISCUSSION THREAD

**Discussion by martonlanga** [suren-atoyan/monaco-react#161] [Show and tell]
Figured it out how to load the local monaco-editor instance using Snowpack. Leaving it here in case someone stumbles upon this.

1. Install `monaco-editor`

2. Mount minified monaco-editor to an url accesible path.
Inside `snowpack.config.js`:
```ts
  mount: {
    // monaco-editor
    'node_modules/monaco-editor/min/vs': {
      url: '/monaco-editor',
      static: true,
      resolve: false,
    },
  },
```

3. Point to the path during init:
```ts
monaco.config({ paths: { vs: 'monaco-editor' } })
```

<img width="519" alt="Screen Shot 2021-01-12 at 20 47 33" src="https://user-images.githubusercontent.com/22799854/104364813-69a59480-5517-11eb-902f-2ae66cf60051.png">

**Comment by suren-atoyan** [comment]
@martonlanga that's great, thank you for your effort 🎉

**Reply by martonlanga** [reply]
@suren-atoyan Just realized that I'm getting these 404 errors in the snowpack dev console.

Tried to mount `min-maps` to `/min-maps` path with no luck. Also, interestingly there are no requests in the network tab for these resources.

Should I try to make them available on a different path, or what am I doing wrong here?

<img width="770" alt="Screen Shot 2021-01-18 at 19 50 44" src="https://user-images.githubusercontent.com/22799854/104953316-75d19c00-59c6-11eb-9ffa-9606887d9d99.png">

**Comment by suren-atoyan** [comment]
@martonlanga hmm, quite strange. Looks like "someone" forces it to download maps from "never-existed" paths. Can you please write down the steps to reproduce your environment?

**Reply by martonlanga** [reply]
Failed to mention the warnings that show up in my console (they also show up when I disable all the DevTools):
<img width="747" alt="Screen Shot 2021-01-19 at 17 04 40" src="https://user-images.githubusercontent.com/22799854/105060222-6c087100-5a78-11eb-9ccb-0f53075c50ea.png">

Whenever I load `monaco-editor` from a CDN, the warnings go away.

Mounting doesn't work because chrome(?) rejects sending `*.map` files. (https://stackoverflow.com/questions/18725600/reference-404-error-for-a-request-for-the-jquery-min-map-file)

Could make them disappear by disabling Javascript Source Maps in my Chromium instance (I'm working on an Electron app btw).
